bigtable/google/cloud/bigtable/table.py

```diff
@@ -17,6 +17,13 @@
 
 import six
 
+from google.api_core.exceptions import RetryError
+from google.api_core.exceptions import Aborted
+from google.api_core.exceptions import DeadlineExceeded
+from google.api_core.exceptions import ServiceUnavailable
+from google.api_core.exceptions import from_grpc_status
```

Comment:
@justinseinlin @lukesneeringer Future reference: these should be in alphabetical order. (We really need a tool to enforce this, so don't feel bad.)

Reply:
:man_facepalming: Thanks for checking and following up with a cleanup PR!

Reply:
Sure thing. (I was a bit intimidated by the size of this PR, so was hoping not to check it. But then I had to write release notes for it, so I had no choice but to read it.)